tensorboard/plugins/pr_curve/pr_curves_plugin.py

```diff
@@ -214,29 +214,42 @@ def is_active(self):
     return any(six.itervalues(all_runs))
 
   def _process_tensor_event(self, event, thresholds):
-    """Converts a TensorEvent into an dict that encapsulates information on it.
+    """Converts a TensorEvent into a dict that encapsulates information on it.
 
     Args:
       event: The TensorEvent to convert.
       thresholds: An array of floats that ranges from 0 to 1 (in that
-        direction).
+        direction). This list of thresholds is not culled. We span the whole
```

Comment:
What does "not culled" mean?

Is "We span the whole range from 0 to 1" not redundant with "An array of floats that ranges from 0 to 1"? If not, please clarify what it actually means.

Reply:
I think this is more concise and clearer.